support/package_nginx

```diff
@@ -99,8 +99,19 @@ pushd /app/vendor/nginx >/dev/null
 tar zcvf "${tempdir}/${nginx_filepath}" .
 popd >/dev/null
 
-s3_upload "$S3_BUCKET" "$nginx_filepath" "${STACK}/${nginx_filepath}"
-"$basedir/manifest" nginx
-"$basedir/package-checksum" "nginx-${nginx_version}"
+
+if ! s3_upload "$S3_BUCKET" "$nginx_filepath" "${STACK}/${nginx_filepath}"; then
+	echo "Unable to upload nginx package to Object Storage!" >&2
+fi
+
+if ! "$basedir/manifest" "nginx"; then
+	echo "Unable to generate/upload manifest for nginx." >&2
+	echo "Something's wrong and needs to be fixed." >&2
+fi
+
+if ! "$basedir/package-checksum" "nginx-${nginx_version}"; then
+	echo "Unable to generate checksums for nginx." >&2
+	echo "Something's wrong and needs to be fixed." >&2
+fi
```

Comment:
question: shouldn't we exit with error in these conditions?

Reply:
Of course :man_facepalming: